### DISCUSSION TITLE
Budibase app loads data to RAM and then crashes

### DISCUSSION THREAD

**Discussion by maiducan** [Budibase/budibase#6577] [Help] [answered]
Hello guys!

My budibase app consumes a lot of RAM when loading data from Budibase DB (the most when fetching relationship data) but it doesn't seem to free it after some time (i.e. website closed).. It stays in the memory till the point it reaches allowed docker/server maximum (I have 6GB) and then bbapp container crashes/restarts and the memory is released. Until this circle happens again (in my case twice a day). I was checking docker stats and using the app at the same time so that i could clearly see it.

I also have another similar app but using postgress db and that is working perfectly/efficiently. 
Is there anybody who has the same issue? 

![image](https://user-images.githubusercontent.com/8984651/177551008-47fecef1-a949-47bf-abd6-0d3288704582.png)

**Comment by Rory-Powell** [comment, marked as answer]
Follow https://github.com/Budibase/budibase/issues/6580 for updates
